newIDE/app/scripts/lib/WikiHelpLink.js

```diff
@@ -1,6 +1,6 @@
 // @ts-check
 
-const gdevelopWikiUrlRoot = 'https://wiki.gdevelop.io/gdevelop5';
+const gdevelopWikiUrlRoot = '/gdevelop5';
```

Comment:
it's used in multiple scripts, are you sure it doesn't break the links?

Reply:
yes, it's the format we used when writing documentation.